### ISSUE TITLE
[Question] Runtime estimation or cutoff

### ISSUE THREAD

**Issue by stheid** [kuhy/glade#1]
Hi,

I hope its ok if i ask a technical question here:

In our project we are trying to use the learned grammar, hence we have not yet worked a lot with the theory on grammar learning, but we where wondering what influences the runtime of the grammar learning. As an experiment i started with learning the grammar of png files. I assume thats a little more challenging than xml but i wonder it there are ways to cutoff the grammar inference and have a look at the intermediate model. Also i am wondering if we are talking about trainingtime of weeks or hours. in this case.

Currently i have let it train for 16h and it still says "learning grammar"

Best, Stefan

**Comment by stheid** [comment]
To be a little more precise: As we are also working in a fuzzing scenario we are not so much interested in a very precise model, however we need to train models for various domains.

**Comment by McSinyx** [comment]
> trainingtime of weeks or hours

Did it finish?  If so, how long did it take?

**Comment by stheid** [comment]
No, we killed it and never really achieved to train it. i think the tool was not suitable for our case
